test/e2e/sources/googlecloudauditlogs/main.go

```diff
@@ -135,7 +140,6 @@ var _ = Describe("Google Cloud AuditLogs source", func() {
 				readReceivedEvents := readReceivedEvents(f.KubeClient, ns, sink.Ref.Name, &receivedEvents)
 
 				Eventually(readReceivedEvents, receiveTimeout, pollInterval).ShouldNot(BeEmpty())
-				Expect(receivedEvents).To(HaveLen(1))
```

Comment:
Wouldn't it be better to expect there to be at least one event received otherwise the test will panic in the next line attempting to obtain the first element of a zero length array?

Reply:
That case is covered by `ShouldNot(BeEmpty())` above. We won't reach that point if no event is received.

Reply:
Oh right. I was expecting a `> 0` check here, but that works too.